ddpui/api/webhook_api.py

```diff
@@ -123,7 +126,11 @@ def post_notification_v1(request):  # pylint: disable=unused-argument
     if send_failure_notifications:
         org = get_org_from_flow_run(flow_run)
         if org:
-            email_flowrun_logs_to_orgusers(org, flow_run["id"])
+            email_flowrun_logs_to_superadmins(org, flow_run["id"])
+            notify_platform_admins(org, flow_run["id"])
+            notify_org_managers(
+                org, "A jobs has failed, please visit https://dashboard.dalgo.in/ for more details"
```

Comment:
do you want to pick the webapp_url from config or `.env` ? Failure notifications on staging will then look weird while testing

Reply:
done